graspy/align/seedless_procrustes.py

```diff
@@ -0,0 +1,373 @@
+# Copyright 2020 NeuroData (http://neurodata.io)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import ot
+import numpy as np
+from sklearn.utils import check_array
+
+from .base import BaseAlign
+from .sign_flips import SignFlips
+from .orthogonal_procrustes import OrthogonalProcrustes
+
+
+class SeedlessProcrustes(BaseAlign):
+    """
+    Implements an algorithm that matches two datasets using an orthogonal
+    matrix. Unlike OrthogonalProcrustes, this does not use a matching between
+    entries. It can even be used in the settings when the two datasets do not
+    have the same number of vertices.
+
+    In essence, it aims to simultaneously obtain a, not necessarily 1-to-1,
```

Comment:
note for someone to write out the math for this (maybe in a `Notes` section). I think at least the math for the objective function will be most clear way to understand the setup

Reply:
(See #441.)

Reply:
i still think a "notes" section that explains what all is going on in math would be good. aside from just the `score_` docs

Reply:
@bdpedigo see https://github.com/microsoft/graspologic/blob/ee0f51607ece50df1dc2c2849347b2dce55efe07/graspy/align/seedless_procrustes.py#L108-L140 . resolved?